applications/nrf_desktop/doc/swift_pair_app.rst

```diff
@@ -36,16 +39,13 @@ To use the Swift Pair module, you must enable the following Kconfig options:
   * :ref:`CONFIG_DESKTOP_SWIFT_PAIR_ADV_GENERAL_PEER <config_desktop_app_options>`
 
   If the dongle peer is disabled, there is no reason to use the module.
-  The Swift Pair advertising data provider can be simply enabled or disabled during a build time through a dedicated Kconfig option.
-
-Set the :ref:`CONFIG_DESKTOP_SWIFT_PAIR <config_desktop_app_options>` Kconfig option to enable the module.
-This option automatically selects the :ref:`CONFIG_DESKTOP_BLE_DONGLE_PEER_ID_INFO <config_desktop_app_options>` Kconfig option to enable the dongle peer ID information event.
+  You can enable or disable the Swift Pair advertising data provider during build time through a dedicated Kconfig option.
```

Comment:
`  You can enable or disable the Swift Pair advertising data provider during build time through a dedicated Kconfig option.` -> This seems not needed here as the option was described a few paragraphs above

Reply:
Removed.